py/pydeck_earthengine_layers/pydeck_earthengine_layers.py

```diff
@@ -12,7 +12,12 @@
 class EarthEngineLayer(pdk.Layer):
     """Wrapper class for using the Earth Engine custom layer with Pydeck
     """
-    def __init__(self, ee_object, credentials=None, **kwargs):
+    def __init__(
+            self,
+            ee_object,
+            credentials=None,
+            library_url=EARTHENGINE_LAYER_BUNDLE_URL,
+            **kwargs):
         """EarthEngineLayer constructor
 
         Args:
```

Comment:
would be nice to add a doc comment about the new argument

Reply:
Yes, also the docstring should be more standardized, certainly if we use Sphinx for docs in the future (though Sphinx might be overkill if we have just one class).